### DISCUSSION TITLE
On/Off PID not displaying

### DISCUSSION THREAD

**Discussion by Strelica12** [agronick/aa-torque#30] [Q&A]
Hi
Seems that PIDs with unit ON / OFF instead of number are not displayed.
In torque they work properly.
Any suggestion please?

**Comment by agronick** [comment]
Go into unit conversions and select boolean value.

**Reply by Strelica12** [reply]
Many thanks for feedback.
Seems there is some other issue as i tried to use proposed solution but without luck. 
It wont update status in AA Torque although in Torque PRO it dispalys status as expected (On or Off) for the tested PID.

PID in question is DPF REGEN STATUS for Mazda 3 BM.
Other dpf pids seems to work fine (ie average distance, current distance etc...) in both AA Torque and Torque Pro.

**Comment by agronick** [comment]
It won't display anything until it gets a value that is not 0.

**Reply by Strelica12** [reply]
Thx again...
Before replying I tested the settings during dpf regen cycle while driving.
Torque PRO was showing DPF REGEN STATUS as ON while AA Torque didn't display status...

I will try to install latest AA Torque RC which was released recently and see if there is any change.

Maybe not question for this thread, but what is mapping.txt in the assets of latest AA Torque RC?

One more time, many thanks for providing this great app.

**Reply by agronick** [reply]
Try doing no conversion and see if you can just get it to give back a 0 or a 1. The mapping is for finding out where errors are happening when people send bug reports.
